providers/azure/resources/azure.lr

```diff
@@ -1697,9 +1697,25 @@ private azure.subscription.cloudDefenderService @defaults("defenderForServers de
   subscriptionId string
   // Whether the monitoring agent is automatically provisioned on new VMs
   monitoringAgentAutoProvision() bool
-  // List of Defender for Server components and whether they are enabled
+  // List of Defender for Servers components and whether they are enabled
```

Comment:
we cannot use boolean indicators for the text, when the output is non dict with multiple values.